api/nodes/views.py

```diff
@@ -1726,3 +1727,31 @@ def perform_create(self, serializer):
         serializer.validated_data['target'] = node
         serializer.validated_data['node'] = node
         serializer.save()
+
+class NodeInstitutionDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, NodeMixin):
+    permission_classes = (
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        AdminOrPublic
+    )
+
+    required_read_scopes = [CoreScopes.NODE_BASE_READ, CoreScopes.INSTITUTION_READ]
+    required_write_scopes = [CoreScopes.NODE_BASE_WRITE]
+    serializer_class = InstitutionDetailSerializer
+
+    model = Institution
+    view_category = 'nodes'
+    view_name = 'node-institution-detail'
+
+    def get_object(self):
+        node = self.get_node()
+        if self.request.method in drf_permissions.SAFE_METHODS:
+            return node.primary_institution
```

Comment:
If there is no primary_institution associated with the Node, the response will be incorrect.  I would raise NotFound.